src/tokens.md

```diff
@@ -337,9 +337,9 @@ b"\\x52"; br"\x52";                  // \x52
 > **<sup>Lexer</sup>**\
 > C_STRING_LITERAL :\
 > &nbsp;&nbsp; `c"` (\
-> &nbsp;&nbsp; &nbsp;&nbsp; ~\[`"` `\` _IsolatedCR_]\
-> &nbsp;&nbsp; &nbsp;&nbsp; | BYTE_ESCAPE\
-> &nbsp;&nbsp; &nbsp;&nbsp; | UNICODE_ESCAPE\
+> &nbsp;&nbsp; &nbsp;&nbsp; ~\[`"` `\` _IsolatedCR_ _ZeroByte_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | BYTE_ESCAPE _except `\0` or `\x00`\
+> &nbsp;&nbsp; &nbsp;&nbsp; | UNICODE_ESCAPE _except `\u{0}`, `\u{00}`, …, `\u{000000}`\
```

Comment:
```suggestion
> &nbsp;&nbsp; &nbsp;&nbsp; | BYTE_ESCAPE <sub>_Except `\0` or `\x00`_</sub>\
> &nbsp;&nbsp; &nbsp;&nbsp; | UNICODE_ESCAPE <sub>_Except `\u{0}`, `\u{00}`, …, `\u{000000}`_</sub>\
```

The italics weren't coming out properly. I've imitated the markup under "Reserved prefixes", which I think is the most similar case.

Reply:
I had omitted the trailing `_` to close the italics. I've added that in instead of using `<sub>`, see what you think?

Reply:
I have no objection to that